src/webgpu/capability_info.ts

```diff
@@ -38,6 +38,34 @@ export const kQueryTypes = keysOf(kQueryTypeInfo);
 /** Required alignment of a GPUBuffer size, by spec. */
 export const kBufferSizeAlignment = 4;
 
+/** Per-GPUBufferUsage type info. */
+export const kBufferUsageTypesInfo: {
+  readonly [name: string]: number;
+} = /* prettier-ignore */ {
+  'map-read':      Number(GPUConst.BufferUsage.MAP_READ),
+  'map-write':     Number(GPUConst.BufferUsage.MAP_WRITE),
+  'index':         Number(GPUConst.BufferUsage.INDEX),
+  'vertex':        Number(GPUConst.BufferUsage.VERTEX),
+  'uniform':       Number(GPUConst.BufferUsage.UNIFORM),
+  'storage':       Number(GPUConst.BufferUsage.STORAGE),
+  'indirect':      Number(GPUConst.BufferUsage.INDIRECT),
+  'query-resolve': Number(GPUConst.BufferUsage.QUERY_RESOLVE),
+};
+/** List of all GPUBufferUsage type values. */
+export const kBufferUsageTypes = keysOf(kBufferUsageTypesInfo);
```

Comment:
mm maybe we can do something like...
```typescript
export const kBufferUsageTypes = Object.keys(GPUConst.BufferUsage) as (keyof typeof GPUConst.BufferUsage)[];

// I think this should work
for (const t of kBufferUsageType) {
  const usage = GPUConst.BufferUsage[t];
}
```

but maybe name it somethng like kBufferUsageKeys or something like that so it's clear it's a key into GPUConst.BufferUsage

Reply:
Played around with it a bit and ended up with this. PTAL and see if there's any better way to do it. (I'm still quite new the this TS typing and the errors I saw are rather cryptic sometimes so not sure if this is the best way.)

Ultimately, the goal is so that we can use a string as the param, and use some sort of mapping in the test to get the flags. A string in the param is a lot easier to read in the CTS format instead of opaque integers.